dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-keycloak-dynamic/dist-dynamic/package.json

```diff
@@ -11,8 +11,8 @@
   "backstage": {
     "role": "backend-plugin-module",
     "supported-versions": "1.32.4",
-    "pluginId": "janus-idp-keycloak-backend",
-    "pluginPackage": "janus-idp-keycloak-backend"
+    "pluginId": "backstage-community-plugin-catalog-backend-module-keycloak",
+    "pluginPackage": "backstage-community-plugin-catalog-backend-module-keycloak"
```

Comment:
are you sure we want `backstage-community-` and not `@backstage-community/` or someting else like here?

https://github.com/janus-idp/backstage-showcase/blob/main/dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-pingidentity-dynamic/dist-dynamic/package.json#L14-L15

Reply:
Yeah, need to make changes to the `package.json` in community-plugins as well

Reply:
https://github.com/backstage/community-plugins/pull/1855 merged. Does that mean we need to bump this to 1.32.6 once it's re-released?